frontend/app/API.js

```diff
@@ -1,63 +1,6 @@
 import { browserHistory } from 'react-router'
 import Raven from 'raven-js'
 
-const parse = {
```

Comment:
I liked the pattern of parse 
because it helped make it visible on the frontend what fields you were getting from the backend
a kind of documentation in code 
... whitelisting  to me meant more typing, but more clarity
but I dont think its a big deal either way